src/birthder.js

```diff
@@ -56,13 +56,23 @@
         "Happy Birthday on your very special day, I hope that you don't die before you eat your cake."
     ];
 
-    // Selects random image URI from a list of images (returned by the search)
+    /**
+     * Selects random image URI from a list of images (returned by the search)
```

Comment:
Third person, singular. However, in other cases imperative form of the verb is used. Choose one particular style and follow it. In this case should be something like
> Select random image...